tools/TweakOpenApi.ps1

```diff
@@ -16,12 +16,12 @@ $prepositionReplacements = @{
     At   = "GraphAPre"
     For  = "GraphFPre"
     Of   = "GraphOPre"
-    Or   = "GraphRPre"
+    #Or   = "GraphRPre" Handling of this preposition needs to be reviewed
```

Comment:
Out of curiosity, is there a specific reason to comment this out?

Reply:
Yes. I noticed operation ids where ``Or`` doesn't appear as a conjunction was being renamed. For example, an operation id that contains "Org" or "Organization". The review will involve detecting appearance of "Or" as a conjuction only.